### ISSUE TITLE
Quality comparison

### ISSUE THREAD

**Issue by OndroNR** [toots/shine#27]
Project is missing quality comparison as faster algorithm with missing Psychoacoustic Model could have lower audio quality at same bitrate.

**Comment by toots** [comment]
I agree with you. However, quality tests can be hard to setup, as far as I know, those tend to be quite subjective. Do you know of any good way to proceed?

**Comment by eblanca** [comment]
It is a time consuming activity indeed, but the more subjects gets involved, the more information you get.
[code listening tests](https://en.wikipedia.org/wiki/Codec_listening_test).
